File: Web/Source/app/prestamos-checklists/shared/modelos/registrar-prestamo-resultado.ts
export class RegistrarPrestamoResultado {
  public idPrestamo: number;
  public numeroPrestamo: number;
  public apellidoNombre: string;
  public cuil: string;

  constructor(idPrestamo?: number,
              numeroPrestamo?: number,
              apellidoNombre?: string,
              cuil?: string) {
    this.idPrestamo = idPrestamo;
    this.numeroPrestamo = numeroPrestamo;
    this.apellidoNombre = apellidoNombre;
    this.cuil = cuil;
  }
}

File: Modulos/Formulario/Formulario.Aplicacion.Consultas/Resultados/RegistrarPrestamoResultado.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Formulario.Aplicacion.Consultas.Resultados
{
    public class RegistrarPrestamoResultado
    {
        public decimal IdPrestamo { get; set; }
        public decimal NumeroPrestamo { get; set; }

        public RegistrarPrestamoResultado()
        {
        }

        public RegistrarPrestamoResultado(decimal idPrestamo, decimal numeroPrestamo)
        {
            IdPrestamo = idPrestamo;
            NumeroPrestamo = numeroPrestamo;
        }
    }
}
